compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -7,7 +7,7 @@ function parseExpr(Tokenizer tok) returns Expr|err:Syntax {
     Position startPos = tok.currentStartPos();
     if t == "[" {
         check tok.advance();
-        Expr[] members = check parseExprList(tok, "]");
+        ExprList {exprs : members} = check parseExprList(tok, "]");
```

Comment:
Do we need ExprList? Can't we use `currentEndPos`?

Reply:
@jclark do you mean whether  `currentEndPos` will give the same as `closeParenPos`? I don't think they'll give the same position.

Reply:
Yes, you're right: currentEndPos would be one more than closeParenPos.

I suggest you make parseExprList return a tuple `[Expr[], Position]`